<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-GOITT5R" failures="0" tests="6" name="sila_test_cases.Check_handle" time="5.033" errors="0" timestamp="2020-12-11T18:38:31 IST" skipped="0">
  <testcase classname="sila_test_cases.Check_handle" name="Test_01_check_with_available_user_handle" time="1.533"/>
  <system-out/>
  <testcase classname="sila_test_cases.Check_handle" name="Test_02_check_with_already_taken_user_handle" time="0.456"/>
  <system-out/>
  <testcase classname="sila_test_cases.Check_handle" name="Test_03_check_with_empty_user_handle" time="0.393"/>
  <system-out/>
  <testcase classname="sila_test_cases.Check_handle" name="Test_04_check_with_invalid_authSignature" time="1.174"/>
  <system-out/>
  <testcase classname="sila_test_cases.Check_handle" name="Test_05_check_with_invalid_handle_pattern" time="0.400"/>
  <system-out/>
  <testcase classname="sila_test_cases.Check_handle" name="Test_06_check_with_invalid_App_handle" time="1.077"/>
  <system-out/>
</testsuite> <!-- sila_test_cases.Check_handle -->
